src/semi_canonicalize.cc

```diff
@@ -297,6 +292,27 @@ bool SemiCanonical::check_fock_matrix() {
     return semi;
 }
 
+void SemiCanonical::set_U_to_identity() {
+    Ua_->identity();
+    Ub_->identity();
+
+    Ua_t_.iterate([&](const std::vector<size_t>& i, double& value) {
+        if (i[0] == i[1]) {
```

Comment:
Can you please shorten this and the next lines to:
`value = (i[0] == i[1]) ? 1.0 : 0.0;`